<?xml version="1.0" ?>
<launch>

  <arg name="config"		default="aist"/>
  <arg name="camera_name"	default="a_phoxi_m_camera"/>
  <arg name="ns"		default="/"/>

  <node ns="$(arg ns)" name="handeye_calibration"
	pkg="aist_handeye_calibration" type="run_calibration.py"
	output="screen">
    <rosparam file="$(find aist_handeye_calibration
		    )/config/$(arg config)/$(arg camera_name).yaml"/>
    <rosparam file="$(find aist_routines)/config/$(arg config)_config.yaml"/>
  </node>

</launch>
